src/coreclr/jit/importer.cpp

```diff
@@ -3554,16 +3554,13 @@ GenTree* Compiler::impImportStaticReadOnlyField(CORINFO_FIELD_HANDLE field, CORI
                     }
                 }
 
-                JITDUMP("success! Optimizing to ASG(struct, 0).");
-                unsigned structTempNum = lvaGrabTemp(true DEBUGARG("folding static ro fld empty struct"));
+                JITDUMP("Success! Optimizing to STORE_LCL_VAR<struct>(0).");
+                unsigned structTempNum = lvaGrabTemp(true DEBUGARG("folding static RO field empty struct"));
```

Comment:
```suggestion
                unsigned structTempNum = lvaGrabTemp(true DEBUGARG("folding static readonly field empty struct"));
```
We spell out readonly below so might as well do it here too.

Reply:
Also there is one more "ro" just below here, can you change that one too?

Reply:
Addressed.